### ISSUE TITLE
Allow searching for a word without selecting it

### ISSUE THREAD

**Issue by ygoe** [NumeriusNegidius/Context-Search#3]
SmartSearch can search for the word that is being clicked on, without first selecting it. This is extremely helpful because most lookups I do are for a single word.

**Comment by NumeriusNegidius** [comment]
This issue depends on #4. It would be really confusing otherwise.

**Comment by ygoe** [comment]
Agreed.

**Comment by NumeriusNegidius** [comment]
Now that #4 is fixed, there is a new issue where the right-clicked word must be handled by a content script. The content script can send text to the background script (where the context menu code lives) but the menu root label cannot be updated in realtime. This results in the label showing the previous text and not the current text as expected. This is the same issue as #19 and thus depends on https://bugzilla.mozilla.org/show_bug.cgi?id=1215376.

**Comment by ygoe** [comment]
@NumeriusNegidius As far as I understand it, fixing the mentioned hyperlink text issue (#19) should also allow fixing this issue, right? Could you please take a look at it again?
